Lib/typing.py

```diff
@@ -1429,11 +1429,16 @@ def name_by_id(user_id: UserId) -> str:
         num = UserId(5) + 1     # type: int
     """
```

Comment:
Please also update the docstring (it still says "returns a dummy function".

Also, Ivan should really review this.

Reply:
@gvanrossum sure thing – but if I may be so bold as to ask, what should we call it now? A “proxy callable class” maybe?